pkg/gator/test/test.go

```diff
@@ -89,7 +102,12 @@ func Test(objs []*unstructured.Unstructured, includeTrace bool) (*GatorResponses
 	responses := &GatorResponses{
 		ByTarget: make(map[string]*GatorResponse),
 	}
-	for _, obj := range objs {
+	for idx, obj := range objs {
+		if templatesOrConstraints[idx] {
```

Comment:
We should not do this. Self-management is an on-cluster operational concern for admission, otherwise self-validating is fair game.

Reply:
The way I parsed this comment is that "it's ok to review objects that may just be a template or constraint; that is, a review of these items may or may not pass given other constraint templates or constraints" -- did I get this right ?

Reply:
yep!

Reply:
updated and reverted changes